packages/e2e-tests/specs/editor/blocks/gallery.test.js

```diff
@@ -52,6 +61,50 @@ describe( 'Gallery', () => {
 		expect( await getEditedPostContent() ).toMatch( regex );
 	} );
 
+	it( 'gallery caption can be edited', async () => {
+		const galleryCaption = 'Tested gallery caption';
+
+		await insertBlock( 'Gallery' );
+		const filename = await upload( '.wp-block-gallery input[type="file"]' );
```

Comment:
Since this tests the whole gallery caption there is no need to get the returned uploaded `filename` and no reason to call `getUploadedFileFigure`.

Reply:
Copy/paste. Sorry... 😅